resolwe_docker_images/common/build.env

```diff
@@ -1,3 +1,3 @@
 BASE_IMAGE=public.ecr.aws/genialis/resolwebio/base:ubuntu-22.04-14112023
 REPOSITORY=resolwebio/common
-TAG=4.1.1
+TAG=4.1.2
```

Comment:
Check the version. CHANGELOG says 4.2.0.

Reply:
This is the question - whether we change it to 4.1.2. or to 4.2.0.